src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -1047,6 +1057,13 @@ private void DrawPatchLinks()
     /// </summary>
     private void RebuildMap()
     {
+        playerSpeciesPopulationIndicators = new List<Control>();
+        foreach (var node in populationIndicatorContainer.GetChildren())
```

Comment:
GetChildren is still here, this needs to go I think GetChildren is a sign of an architecture problem. Use a persistent list for knowing all the existing indicators that way you can also avoid the type cast on line 1063.